ClassHierarchyAndCollections/ClassHierarchyAndCollections/Business.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClassHierarchyAndCollections
+{
+    /// <summary>
+    /// An orginization based around the selling of goods or services in exchange for money.
+    /// </summary>
+    public class Business : Organization
+    {
+
+        /// <summary>
+        /// The Chief Executive Officer of the Business.
+        /// </summary>
+        public Individual ChiefExecutiveOfficer { get; set; }
+
+        /// <summary>
+        /// The List of all Employees in the Business.
+        /// </summary>
+        public List<Individual> Employees { get; set; }
```

Comment:
Typically, collection properties should be read-only (no set method) to protect your internal state. You can allow callers to get the list and therefore add and remove items from the list, but you typically would not allow callers to stomp on the existing list contents by assigning a new list to the property.